sig-windows/charter.md

```diff
@@ -20,6 +20,7 @@ Windows specific implementation.
 #### Cross-cutting and Externally Facing Processes
 
 - Work with other SIGs on areas where Windows and Linux (and possibly other OSes in the future) deviate from one another in terms of functionality.
+- Providing community guidance for external tools and paradigms necessary for orchestrating the windows stack, including `wins`, `csi-proxy`, and the umbrella (windows privileged containers)[https://github.com/ambguo/enhancements/blob/master/keps/sig-windows/1981-windows-privileged-container-support/20201981-windows-privileged-container-support.md] initiative.
```

Comment:
I'm not sure it is a good idea to call our specific tools here (especially wins where usage is contentious due to its potential for bypassing a lot of security boundaries).

Reply:
updated